### ISSUE TITLE
The process function does not return the packet type processed

### ISSUE THREAD

**Issue by MonitorComputing** [madleech/ArduinoCMRI#11]
Despite what the README and function documentation in the cpp source file says the process function simply returns a bool, true for "poll" or "set" otherwise false. That's a shame as I have an application where if no "poll" is processed I'd like to trigger a transmit to cause an immediate update.

**Comment by madleech** [comment]
Agreed this is an oversight. I've created #27 to address. This will be followed by a minor version bump as it's technically changing the behaviour. I will try and merge #26 first though to address the bug there.
